setup.py

```diff
@@ -10,7 +10,12 @@
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
 setup(name='mwclient',
-      version='0.10.1',  # Use bumpversion to update
+      # MAINTAINERS: to cut a release, ensure CHANGELOG.md is updated,
+      # then run `bump-my-version bump major|minor|patch`
+      # (as appropriate), check the commit looks correct and is tagged
+      # vX.Y.Z, and push. the.github/workflows/release.yml action will
+      # publish to PyPi
```

Comment:
This is so close to semantic line-breaking — can we re-wrap it to get it all the way there? 😁

```suggestion
      # MAINTAINERS: to cut a release, ensure CHANGELOG.md is updated,
      # then run `bump-my-version bump major|minor|patch` (as suitable),
      # check the commit looks correct and is tagged vX.Y.Z, and push.
      # The .github/workflows/release.yml action will publish to PyPI
```

Reply:
Btw, I think this should be documented [here](https://github.com/mwclient/mwclient/blob/master/docs/source/development/index.rst). Either the text should be duplicated there, or this comment should link to the [resulting page](https://mwclient.readthedocs.io/en/latest/development/) (ideally with a section anchor).

Reply:
Eh, I'm not sure. That feels more like a "contributors' guide" than a "maintainers' guide" to me. I felt like this was more likely to be somewhere a future maintainer might actually look when trying to do an update.

Reply:
...but hey, if you insist, how about this? (yes, readthedocs is somehow still getting automatically updated, not sure how, but it is!)

Reply:
I see what you mean. I do agree with the way you did it, though — after all, we want to (1) ensure all processes are well documented, for both contributors and maintainers; and (2) provide a smooth progression path from one-time/occasional contributor to regular contributor to maintainer.

I've left a few minor edit suggestions as inline comments, but other than that, LGTM :)